packages/inter-protocol/test/vaultFactory/snapshots/test-vaultFactory.js.md

```diff
@@ -185,6 +185,7 @@ Generated by [AVA](https://avajs.dev).
             brand: Object @Alleged: IST brand {},
             value: 0n,
           },
+          lockedQuote: undefined,
```

Comment:
surprised if this doesn't need updating

Reply:
Good catch!  I also missed a `test.only` that I added.  both fixed.

Reply:
ah, I didn't see the `test.only`. reminds me we need https://github.com/Agoric/agoric-sdk/issues/4941